scripts/smoke/local-runtime-smoke.ps1

```
@@ -0,0 +1,316 @@
+    # Wait for qdrant
+    $qdReady = $false
+    for ($i = 0; $i -lt 30; $i++) {
+        Start-Sleep -Seconds 2
+        try {
+            $resp = Invoke-WebRequest -Uri "http://localhost:6333/healthz" -TimeoutSec 3 -UseBasicParsing -ErrorAction SilentlyContinue
+            if ($resp.StatusCode -eq 200) {
+                Write-SmokeLog "Qdrant is ready"
+                $qdReady = $true
+                break
+            }
+        } catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)